### DISCUSSION TITLE
State of Rhodes in 2024?

### DISCUSSION THREAD

**Discussion by HighPriest** [rhomobile/rhodes#1125] [General]
Hi.
I have stumbled upon the rhodes mobile project through work history of a friend of my. I haven't heard before about this project, despite its apparent high advancement. And, from what I was able to find on the internet, at some point in time it had quite alot of fully functional prototypes made!

So, as the project has not been viral at any point in time (even youtube videos presenting working solutions dont have many views) and even the documentation links in the main README of this project lead to dead ends.
Is this project worth investing into right now?
Why is it worth investing into?
How far is this project from facilitating creation of modern, performant, cross-platform applications?

Thank you and best wishes to the team still poking on this code <3

**Comment by dmitrys** [comment]
Hi,
Thank you.
Few members of original Rhomobile startup still support RhoMobile project. Rhomobile was acquired by Motorola and will part of Zebra. From 2016 Rhomobile become fully open-source and supported by Tau Technologies company.
Zebra press-release - https://investors.zebra.com/news-and-events/news/news-details/2016/Zebra-RhoMobile-App-Development-Now-Open-Source/default.aspx
Now we continue support RhoMobile and has set of clients who use RhoMobile for app development.
Latest release is 7.6 and can be downloaded from https://rhomobile.tau-platform.com/index.html
Documentation - http://docs.tau-platform.com/en/7.5/home
RhoMobile still fully open-source and support latest versions of iOS and Android.
Our company Tau Technologies has another products, for example - enterprise browser for mobile devices - RhoBrowser (https://files.tau-platform.com/Browser/Tau_Browser_ENG.pdf). Our company also engages in custom development, consulting.

If you ask about why RhoMobile should be used for enterprise mobile app development, I will answer you:
Web technologies is most popular, but solutions like PhoneGap/Cordova not useful for complex heavy applications. So RhoMobile solve all problem - RhoMobile propose WebView only for UI but logic and data processing work in separate thread in very comfortable environment (like Ruby on Rails) where developers can use many already exists ruby code because ruby still popular for server size data processing solutions.
So shortly, RhoMobile is a Electron for mobile platforms but based on ruby instead of Javascript (Node.js).
Other new solutions like Flutter require special developers and require learn it complicated technology, but RhoMobile require html/js/ruby developers -just frontend developers and backend developers without any additional requirements.

**Reply by HighPriest** [reply]
Thank you for extensive answer

**Reply by solerman** [reply]
Also it is very fast for offline apps. We couldn't match its performance using React native for example, even with custom sql (which requires one project for each platform). Anything Flutter-related is in hold due to Googles firing most of the flutter team, microsoft killed linux support in it's new ui platform, so anthing .net might be on hold too (I mean, you might build to android from linux, but performance there is slow. Dunno about iphone). That leaves only Kotlin multiplatform, still new
